website/assets/package.json

```diff
@@ -10,22 +10,22 @@
     "@hashicorp/hashi-alert": "^1.1.0",
     "@hashicorp/hashi-button": "1.0.5",
     "@hashicorp/hashi-callouts": "^2.0.5",
-    "@hashicorp/hashi-content": "^0.0.7",
-    "@hashicorp/hashi-docs-sidenav": "^0.1.3",
-    "@hashicorp/hashi-docs-sitemap": "^0.1.3",
+    "@hashicorp/hashi-content": "^0.0.8",
+    "@hashicorp/hashi-docs-sidenav": "^0.1.4",
+    "@hashicorp/hashi-docs-sitemap": "^0.1.4",
     "@hashicorp/hashi-footer": "1.0.6",
     "@hashicorp/hashi-ga-form-fields": "1.0.2",
-    "@hashicorp/hashi-global-styles": "^1.0.8",
-    "@hashicorp/hashi-hero": "^2.1.6",
-    "@hashicorp/hashi-image": "1.0.4",
+    "@hashicorp/hashi-global-styles": "^1.0.9",
+    "@hashicorp/hashi-hero": "^2.1.9",
+    "@hashicorp/hashi-image": "1.0.5",
```

Comment:
However, we're never actually exporting image ourselves.  It's simply there because it's a peer dependency being used by hero and downloads (I think it's just those two).  We don't export it and use it anywhere ourselves.

Please tell me if I'm misunderstanding something here.

Reply:
Ah in that case we don't need to add it to reshape then right? That only rehydrates top level elements

Reply:
Right.  It's just a peer dep.  We aren't exporting in `reshape.js`.